sources/apigateway/resource.go

```diff
@@ -0,0 +1,97 @@
+package apigateway
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/service/apigateway"
+	"github.com/aws/aws-sdk-go-v2/service/apigateway/types"
+	"github.com/overmindtech/aws-source/sources"
+	"github.com/overmindtech/sdp-go"
+)
+
+func convertGetResourceOutputToResource(output *apigateway.GetResourceOutput) *types.Resource {
+	return &types.Resource{
+		Id:              output.Id,
+		ParentId:        output.ParentId,
+		Path:            output.Path,
+		PathPart:        output.PathPart,
+		ResourceMethods: output.ResourceMethods,
+	}
+}
+
+func resourceOutputMapper(scope string, awsItem *types.Resource) (*sdp.Item, error) {
+	attributes, err := sources.ToAttributesCase(awsItem, "tags")
+	if err != nil {
+		return nil, err
+	}
+
+	item := sdp.Item{
+		Type:            "apigateway-resource",
+		UniqueAttribute: "id",
+		Attributes:      attributes,
+		Scope:           scope,
+	}
+
+	// This is used in integration tests.
+	// If we get this far, we can assume that the item is healthy.
+	item.Health = sdp.Health_HEALTH_OK.Enum()
```

Comment:
Does it make sense to be *always* healthy?

Reply:
Not really, this was for integration test, but I've added a new function that does not depend on source health. So, I can change it.